react_app/package.json

```diff
@@ -8,6 +8,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "passport": "^0.4.1",
+    "prop-types": "^15.7.2",
```

Comment:
is prop-types needed?

Reply:
Not for now but it will be for the next DB integration PR - I'll still remove it for this PR though to keep things clean.